menu.js

```diff
@@ -300,3 +309,38 @@ module.exports.setApplicationMenu = (win) => {
 	const menu = Menu.buildFromTemplate(menuTemplate);
 	Menu.setApplicationMenu(menu);
 };
+
+const iconPath = path.join(__dirname, "assets", "youtube-music-tray.png");
+const example = "Example: 'socks5://127.0.0.1:9999'";
+function setProxy(item, win) {
+	let options = {
+		title: 'Set Proxy',
+		label: 'Enter Proxy Address: (leave empty to disable)',
+		value: config.get("options.proxy") || example,
```

Comment:
Idea of feature for the prompt plugin: the ability to have a placeholder (would be better suited for `example` instead of putting it as value and checking later that `value !== example`)

Reply:
If you are talking about changing the source `custom-electron-prompt`:
Right now `value` is the placeholder
The problem with your idea is that a placeholder could very often be a valid value
for example all keybind options, and counter options starts by displaying the current option and we want to get the correct value even if unchanged

So I think its better to let people handle special cases themselves. 

Unless I misunderstood what you said 😅

Reply:
Yes, idea was to add a implement the placeholder logic/field in custom-electron-prompt! (main pro of a placeholder is to avoid having a fake value to remove when filling the field, contrary to a default value - here, it feels we have a placeholder used a default value 🙃)
Not a big deal though, does not block merging - just an improvement idea for the custom-prompt package :)

Reply:
355f611 changed the input field "placeholder" attribute to the previous example as you suggested
(didn't need to change custom-electron-prompt itself)

I'm not sure its better though, since you can't copy paste or edit that text, and it disappears when typing. which can makes it harder for people that don't know the format